examples/with-docker/packages/eslint-config-custom-server/index.js

```diff
@@ -7,6 +7,9 @@ module.exports = {
   parserOptions: {
     ecmaVersion: "latest",
     sourceType: "module",
+    babelOptions: {
+      presets: [require.resolve('next/babel')],
```

Comment:
One more, this config shouldn't be required here since we're not extending from next for this config.